apps/meteor/ee/app/livechat-enterprise/server/api/lib/outbound.ts

```diff
@@ -0,0 +1,26 @@
+import { ProviderTypeEnum } from '@rocket.chat/core-typings';
+import type { IOutboundProvider, OutboundProviderTypes as ValidProviderTypes } from '@rocket.chat/core-typings';
+
+import { OutboundMessageProvider } from '../../../../../../server/lib/OutboundMessageProvider';
+
+export class OutboundMessageProviderService {
+	private readonly provider: OutboundMessageProvider;
+
+	constructor() {
+		this.provider = new OutboundMessageProvider();
+	}
+
+	private isProviderValid(type?: string): type is ValidProviderTypes {
```

Comment:
You don't need type to be undefined here. `isProviderValid` is only called after checking type is not undefined.